### DISCUSSION TITLE
Recreation.gov now offers campsite availability alerts natively

### DISCUSSION THREAD

**Discussion by BHSPitMonkey** [juftin/camply#301] [General]
[Documentation](https://recreationonestopprod.servicenowservices.com/external?id=kb_article&sys_id=54f0fbdb1b5fe510375587bae54bcb72)

> * Availability Alerts can only be set if the location is already reserved.
> * Only 3 alerts can be set at one time.
> * Availability alerts will send you an email if a booked campsite matching your preferences becomes available. This does not guarantee you a reservation as this alert will be sent to everyone who has the alerts set for the same date.
> * Alerts can only be set for the campground; alerts will not be able to be set for a specific site.
> * If specific amenities are needed, be sure to add these filters before setting an alert.

It's more limited than what you can achieve through Camply (e.g. watching more than 3 campgrounds, more flexible notification options) and it remains to be seen how quick it is (I imagine Recreation.gov will have to burn down a long list of subscribers for very popular locations, and there will probably be more of a delay the later you are in whatever order those get processed in), but this seems like a step in the right direction.

**Comment by juftin** [comment]
Oh interesting! This is awesome to see. It'll be especially interesting to see what happens to some services out there who are monetizing their reverse-engineering of recreation.gov's API.
